apps/events/views.py

```diff
@@ -390,6 +390,26 @@ class CompanyEventViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin, mi
     permission_classes = (AllowAny,)
 
 
+def validate_attend_params(rfid, username):
```

Comment:
Wouldn't this work better as a method on AttendViewSet?

Reply:
There's nothing wrong by adding it to the viewset, but I don't see any difference in doing so. It could potentially in theory be reused other places if needed (e.g. other (future) regme-endpoints, nibble, ...)

Reply:
Sounds like YAGNI, but meh. Ække så nøye.